packages/forklift-console-plugin/src/modules/NetworkMaps/EmptyStateNetworkMaps.tsx

```diff
@@ -24,7 +24,15 @@ const EmptyStatePlans: React.FC<{ namespace: string }> = ({ namespace }) => {
   return (
     <ForkliftEmptyState
       icon={AutomationIcon}
-      title={t('No NetworkMaps found.')}
+      title={
+        namespace ? (
+          <Trans t={t} ns="plugin__forklift-console-plugin">
+            No NetworkMaps found in <strong>{namespace}</strong> namespace.
+          </Trans>
```

Comment:
I keep going between what you have:
  -  "No XXX found in **sample-ns** namespace."

and the other ways to say it:
  - "No XXX found in the **sample-ns** namespace."
  - "No XXX found in namespace **sample-ns**."  :arrow_left: (this one is my current favorite)

Reply:
ok, "No XXX found in namespace **sample-ns**." it is :-)